fido-device-onboard.spec

```diff
@@ -10,9 +10,8 @@ License:        BSD-3-Clause
 
 URL:            https://github.com/fdo-rs/fido-device-onboard-rs
 Source0:        %{url}/archive/v%{version}/%{name}-rs-%{version}.tar.gz
-# See make-vendored-tarfile.sh in upstream repo
-Source1:        %{name}-rs-%{version}-vendor-patched.tar.xz
-Patch1:         0001-Revert-chore-use-git-fork-for-aws-nitro-enclaves-cos.patch
+# We need the vendor tar file for CentOS and RHEL builds
+#Source1:        %%{name}-rs-%%{version}-vendor.tar.xz
```

Comment:
I guess we can just remove this right?

Reply:
No, currently we cannot, the targets related to rpm/srpm creation and the `packit-create-vendor` target modify exactly that line in case of CentOS/RHEL builds to add the vendor tar file. I kept the commented line because it's easier to replace (and because it gives some information). If you want to get rid of it I can modify the Makefile to add the `Source1` line after `Source0` instead of replacing/uncommenting the existing line. It's quite simple actually.

Reply:
yeah, uhm, I mean, the whole point is to _not_ conditionalize the sources in the spec file (similarly to the guidelines https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_do_not_conditionalize_sources) so that we can just have one spec file shared between RHEL/centos-stream/Fedora - it seems this is instead going in a different direction :thinking: maybe propose-downstream (pre-sync) should instead modify the spec to include Source1 for centos-stream and we have that conditional only there

Reply:
Actually, it's possible and we can share the same SPEC file. In fact it will simplify things. The only downside is that we will need to upload the vendor tar file to Fedora ccache even when the vendor's source file is not needed.

Reply:
yep, that downside if perfectly acceptable according to the guidelines (as I read them, correct me if I'm wrong)